client/impl/src/main/java/com/alipay/sofa/registry/client/provider/SubClientConnection.java

```diff
@@ -0,0 +1,33 @@
+package com.alipay.sofa.registry.client.provider;
+
+import com.alipay.remoting.Connection;
+import com.alipay.remoting.ConnectionEventProcessor;
+import com.alipay.remoting.ConnectionEventType;
+import com.alipay.remoting.rpc.protocol.UserProcessor;
+import com.alipay.sofa.registry.client.api.RegistryClientConfig;
+import com.alipay.sofa.registry.client.remoting.ClientConnection;
+import com.alipay.sofa.registry.client.remoting.ServerManager;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author liqiuliang
+ * @create 2022-09-30 20:08
+ */
+public class SubClientConnection extends ClientConnection {
```

Comment:
when connect is fine, it will not reconnect now.
i think we should scheduler check if server nodes has change, and reconnect the sub connection.